beacon-chain/rpc/validator/server.go

```diff
@@ -102,6 +102,49 @@ func (vs *Server) ValidatorIndex(ctx context.Context, req *pb.ValidatorIndexRequ
 	return &pb.ValidatorIndexResponse{Index: index}, nil
 }
 
+// ExitedValidators queries validator statuses for a give list of validators
+// and returns a filtered list of validator keys that are exited.
+func (vs *Server) ExitedValidators(
```

Comment:
Is this supposed to be here?

Reply:
Yep! This happened because some stuff was moved in the file and then moved back but not quite at the same location lol